notebooks/108-gpu-device/108-gpu-device.ipynb

```diff
@@ -138,7 +138,9 @@
    "metadata": {},
```

Comment:
<p>it's not saved to&nbsp;<code>../model/ir_model</code>, but to&nbsp;<code>model/ir_model</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/865/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>